segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/ReadResult.java

```diff
@@ -23,48 +23,57 @@
- * return contiguous data.
- * <li> next() will return a specific instance of ReadResultEntry, depending on where the data to be read is located.
+ * return contiguous data. </li>
+ * <li> next() will return a specific instance of ReadResultEntry, depending on where the data to be read is located. </li>
  * <li> If next() returns an entry that has "isEndOfStreamSegment()" returning true, it means the Read Result has reached
- * the end of a sealed stream, and subsequent calls to hasNext() will return false.
+ * the end of a sealed stream, and subsequent calls to hasNext() will return false. </li>
  * <li> If the data is readily available in memory, the returned ReadResultEntry will contain an already completed future,
- * ready for consumption (using the regular CompletableFuture methods).
+ * ready for consumption (using the regular CompletableFuture methods). </li>
  * <li> If the data is not available in memory (currently), the returned ReadResultEntry will contain a CompletableFuture
  * that will be completed when data becomes available (pulled from Storage or Future Read). If the process of retrieving
- * data fails, the CompletableFuture will fail with the source exception as cause.
+ * data fails, the CompletableFuture will fail with the source exception as cause.</li>
  * <li> If the data requested is beyond the last offset of the StreamSegment (Future Read), the returned CompletableFuture
  * will complete when at least 1 byte of data from that offset is available, or will fail with a TimeoutException when the
- * timeout expires or StreamSegmentSealedException if the StreamSegment has been sealed at or before its offset.
+ * timeout expires or StreamSegmentSealedException if the StreamSegment has been sealed at or before its offset. </li>
  * </ul>
  */
 public interface ReadResult extends Iterator<ReadResultEntry>, AutoCloseable {
     /**
      * Gets a value indicating the Offset within the StreamSegment where this ReadResult starts at.
+     *
+     * @return  offset where the read result started at as a long
      */
     long getStreamSegmentStartOffset();
 
     /**
      * Gets a value indicating the maximum length that this read result can have.
+     *
+     * @return the max length possible for the read result
      */
     int getMaxResultLength();
 
     /**
      * Gets a value indicating the number of bytes that have been consumed via the next() method invocations.
      * Note that this does not track the individual consumption within the objects returned by next().
+     *
+     * @return number of bytes consumed via the next method invocation
+     *
      */
     int getConsumedLength();
 
     /**
      * Gets a value indicating whether this ReadResult is fully consumed (either because it was read in its entirety
      * or because it was closed externally).
+     *
+     * @return true if ReadResult  is fully consumed or closed externally, otherwise false
```

Comment:
There is an extra space after `ReadResult`.

Reply:
space removed